storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/AbstractJdbcQueryTemplater.java

```diff
@@ -20,20 +20,23 @@
 import feast.proto.core.FeatureSetProto;
 import feast.proto.serving.ServingAPIProto;
 import feast.storage.api.retriever.FeatureSetRequest;
-import feast.storage.connectors.jdbc.connection.JdbcConnectionProvider;
+import feast.storage.connectors.jdbc.snowflake.TimestampLimits;
 import io.grpc.Status;
 import java.io.IOException;
 import java.sql.*;
 import java.util.*;
 import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.springframework.jdbc.core.JdbcTemplate;
 
 public abstract class AbstractJdbcQueryTemplater implements JdbcQueryTemplater {
+  private static final Logger log =
+      org.slf4j.LoggerFactory.getLogger(AbstractJdbcQueryTemplater.class);
```

Comment:
I'm getting this error msg 
`ERROR StatusLogger Log4j2 could not find a logging implementation. Please add log4j-core to the classpath. Using SimpleLogger to log to the console...
`
It seems like there is a log4j2 version issue?